### PR TITLE
Add method to get the current quorum size

### PR DESCRIPTION
**Pull request by andresilva**
This PR adds a method to get the current quorum size.

---

This change is [Reviewable](https://reviewable.io/reviews/maidsafe/accumulator/65)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
Thanks for the pull request, and welcome! The MaidSafe team is excited to review your changes, and you should hear from @dirvine (or someone else) soon.

If any changes to this PR are deemed necessary, please add them as extra commits. This ensures that the reviewer can see what has changed since they last reviewed the code. The way GitHub handles out-of-date commits, this should also make it reasonably obvious what issues have or haven't been addressed. Large or tricky changes may require several passes of review and changes.

Please see [CONTRIBUTOR.md](../blob/master/CONTRIBUTOR) for more information.

File: src/lib.rs [hunk 1]
```diff
@@ -139,6 +139,11 @@ impl<Key: PartialOrd + Ord + Clone, Value: Clone + Eq + Hash> Accumulator<Key, V
     pub fn set_quorum_size(&mut self, new_size: usize) {
         self.quorum = new_size;
     }
+
+    /// Returns the current value for `quorum`.
+    pub fn get_quorum_size(&self) -> usize {
```
Comment (dirvine) [inline src/lib.rs]: Hi, thanks for the PR, really nice. I wonder though if it is more idiomatic to call this method (getter) just `quorum_size`. I will ask @afck  what he thinks as well. I generally use set for setters and the item name alone for getters.

**Comment by afck** [comment]
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_[src/lib.rs, line 144 [r1]](https://reviewable.io:443/reviews/maidsafe/accumulator/65#-KNYhx7CJ_bF26W88ME5-r1-144:-KN_tUnnSzfdcCjF_yjr:793121795) ([raw file](https://github.com/maidsafe/accumulator/blob/2ef2a6104f71b8353d08255cd382a9329d912ce5/src/lib.rs#L144)):_
<details><summary>

<i>Previously, dirvine (David Irvine) wrote…</i></summary>



> Hi, thanks for the PR, really nice. I wonder though if it is more idiomatic to call this method (getter) just `quorum_size`. I will ask @afck  what he thinks as well. I generally use set for setters and the item name alone for getters. 
> </details>
> Yes, that's the recommended style: https://github.com/rust-lang/rust/tree/master/src/doc/style/style/naming
> If the field is called `quorum`, maybe the getter should even have _that_ name instead of `quorum_size`? In any case, I'd omit the `get_`.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/maidsafe/accumulator/65)_

**Comment by dirvine** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_[src/lib.rs, line 144 [r1]](https://reviewable.io:443/reviews/maidsafe/accumulator/65#-KNYhx7CJ_bF26W88ME5-r1-144:-KNaTYwa9PalyeoebjL-:997448086) ([raw file](https://github.com/maidsafe/accumulator/blob/2ef2a6104f71b8353d08255cd382a9329d912ce5/src/lib.rs#L144)):_
<details><summary>

<i>Previously, afck (Andreas Fackler) wrote…</i></summary>



> Yes, that's the recommended style: https://github.com/rust-lang/rust/tree/master/src/doc/style/style/naming
> If the field is called `quorum`, maybe the getter should even have _that_ name instead of `quorum_size`? In any case, I'd omit the `get_`.
> </details>
> Excellent, so this small change (rename to quorum) @andrebeat  and we look good to go. Thanks again 

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/maidsafe/accumulator/65)_

**Comment by andresilva** [comment]
For consistency, should I also rename `set_quorum_size` to `set_quorum`?

**Comment by afck** [comment]
Good catch! Yes, please. :+1: 

---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/maidsafe/accumulator/65#-:-KNacTsbU6VIPqZp8DLl:-158111637)_

**Comment by afck** [comment]
Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/maidsafe/accumulator/65)_

**Pull request merged**